pkg/web/apps/apps_http_handler.go

```diff
@@ -32,3 +33,26 @@ func (a *HTTPHandler) GetApps(c echo.Context) error {
 
 	return c.JSON(http.StatusOK, apps)
 }
+
+// GetAppById returns apps by id as JSON from the AppService
+func (a *HTTPHandler) GetAppById(c echo.Context) error {
+
+	id := c.Param("id")
+	if !IsValidUUID(id) {
+		return httperrors.BadRequest(c, "Invalid id supplied")
+	}
+
+	apps, errMessage := a.Service.GetAppById(id)
```

Comment:
I think the standard variable name everywhere seems to be `err` so maybe we should keep it this way.

Reply:
This is a string not a error, May change when I integrate Davids PR